pyproject.toml

```diff
@@ -1,13 +1,12 @@
-[tool.pytest.ini_options]
-addopts = [
-    "--color=yes",
-    "--import-mode=importlib",
-    "--verbose",
-]
-testpaths = [
-    "tests",
+[tool.mypy]
+exclude = [
+    'tests\/data\/test_package_generation\/src\/cookiecutter_test\/__init__\.py',
```

Comment:
Can someone work out how I can get this to work? mypy doesn't seem to be ignoring the file 😢

Reply:
So I encountered this problem yesterday (along with @alessandrofelder) https://github.com/python/mypy/issues/17977. It's not ideal, but maybe we just exclude from `pre-commit`?